src/styles/blocks/container.scss

```diff
@@ -0,0 +1,23 @@
+.container {
+  display: flex;
+  align-items: center;
+  width: 944px;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
```

Comment:
add `margin: 0 auto` to the container to center it horizontally inside main block